crates/meilisearch/tests/index/get_index.rs

```diff
@@ -26,13 +26,13 @@ async fn create_and_get_index() {
 
 #[actix_rt::test]
 async fn error_get_unexisting_index() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
 
     let (response, code) = index.get().await;
 
     let expected_response = json!({
-        "message": "Index `test` not found.",
+        "message": format!("Index `{}` not found.", index.uid),
```

Comment:
If we can do the previous comment, then we can hardcode the index name here:
```suggestion
        "message": "Index `DOES_NOT_EXISTS` not found.",
```